akka-docs/rst/java/persistence.rst

```diff
@@ -503,6 +503,53 @@ not accept more messages and it will throw ``AtLeastOnceDelivery.MaxUnconfirmedM
 The default value can be configured with the ``akka.persistence.at-least-once-delivery.max-unconfirmed-messages``
 configuration key. The method can be overridden by implementation classes to return non-default values.
 
+.. _event-adapters-java:
+
+Event Adapters
+==============
+
+.. note::
+
+  Complete documentation featuring use-cases and implementation examples for this feature will follow shortly.
+
+In long running projects using event sourcing sometimes the need arrises to detach the data model from the domain model
+completely.
+
+Event Adapters help in situations where:
+
+- **Upcasting** – existing events stored in *Version 1* should be "upcasted" to a new *Version 2* representation,
```

Comment:
Ok, how about we call it "model migrations" here? I'd like to add the word upcasting in something like "This technique is sometimes refered to as 'upcasting' in some other libraries"? In case someone familiar with Axon or something like it tries to find "how do I do upcasting in akka?"